### ISSUE TITLE
Sender know how to go back to edit template from review page before sending a message

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#327]
# Description

As a Sender using the manual sending process, I need to be able to know how to go back to edit my template from the review page so that I can fix mistakes I catch when I review my message before sending. 

WHY are we building?
[Participant in Usability testing wasn't confident how to edit their template from the review screen](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwEUFgBd0KLqH7oo/rec6N6URowxp6tFDf?blocks=hide)
[Probably won't be very clear how to modify a message.](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/rect965FYCFaohqeb?blocks=hide)

WHAT are we building?
Improve navigation on the review page, the last step before a Sender is ready to send a message(s)

VALUE created by our solution
Better navigation for senders and more accessible (must be tested for any new a11y issues that may be introduced)

# Acceptance Criteria** (Definition of done) 

- [x]  Figma design of improved navigation of review page  
 
- [ ] Designs updated in English and French

- [ ] Team review

- [ ]  Design critiques (a11y)

- [ ] Link research insight back to hypothesis from Epic or Objective in Airtable

- [ ] Once change/fix/feature is designed, link insight to design artifacts (Figma) in Airtable

Given A sender gets to the "review" page of their final message before sending, when they find a mistake and need to made an edit, then they can navigate back to the template to edit it.

**Comment by amazingphilippe** [comment]
I don't think we fixed this case in particular. Looking back at the quote in airtable, does this still feel like something we need to do?

**Comment by yaelberger-commits** [comment]
I do agree with @adriannelee that we should look to fix this

**Comment by adriannelee** [comment]
Yea we should've attached a screenshot from the start 😅

It's this page. I'm curious why it'd be a stretch? I was thinking of like a 'Edit template' button somewhere near the message text.
![image](https://user-images.githubusercontent.com/49994378/170358652-cd7cab61-83bf-48a0-94e3-85a292ba6d9e.png)

**Comment by amazingphilippe** [comment]
Here are a few additional considerations we'll need to think of:

1. What is the path back from editing a template to the review page
2. What happens when they remove a placeholder variable
3. How would the user know where the mistake is coming from? Is it from the template, spreadsheet, or the placeholder values they've entered manually.

**Comment by amazingphilippe** [comment]
I'm wondering if breadcrumbs for the sending flow might be something worth exploring? To navigate back precisely within that flow. To me it would also solve more navigation issues. 

I think we should do some design iteration and testing on this. 

Also I would remove the a11y label?

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/327?planning-poker) with ZenHub @amazingphilippe @adriannelee @YedidaZalik

**Comment by yaelberger-commits** [comment]
Bring forward on a Wednesday for story definition

**Comment by yaelberger-commits** [comment]
Bring forward on a Wednesday for story definition

**Comment by yaelberger-commits** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/327?planning-poker) with ZenHub @jimleroyer

**Comment by amazingphilippe** [comment]
Just realized I never linked the protoypes created for this: https://www.figma.com/file/fRVRKm7cM8MUuBWUJbrfgU/Sending-from-web-interface-%7C-Envoyer-par-interface-web?node-id=5276%3A22107

**Comment by yaelberger-commits** [comment]
DECISION: This step in the flow is to check variables, sending list, CSV and there are better and mutiple opportunities before getting to this stage where users can more easily edit their template. Recommend iceboxing.

**Comment by yaelberger-commits** [comment]
@adriannelee Can I get your review on the decision to Icebox this card please? See comment above
